apps/server/src/utils/customFields.ts

```diff
@@ -0,0 +1,38 @@
+import { CustomFieldDefinitions, CustomInfo } from 'ontime-types';
+import { isAlphanumeric } from 'ontime-utils';
+import { DataProvider } from '../classes/data-provider/DataProvider.js';
+
+export const createCustomField = async (label: string, field: CustomInfo) => {
```

Comment:
we would need some tests for this functions.
the logic can also be simplified here

It is OK with me that we postpone doing it until we decide on the datashape